core/src/main/java/org/opensearch/sql/analysis/Analyzer.java

```diff
@@ -558,6 +559,29 @@ public LogicalPlan visitAD(AD node, AnalysisContext context) {
     return new LogicalAD(child, options);
   }
 
+  /** Build {@link LogicalAD} for fillnull command. */
```

Comment:
This should link to LogicalEval instead of LogicalAD.

Reply:
Updated.